README.md

```diff
@@ -50,6 +50,29 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
 * (optional) to enable fedora internal audit capability, edit `install_scripts/config` and change the FEDORA_AUDIT variable to true. The FEDORA_AUDIT_LOCATION can also be changed from its default "/audit", if necessary.
 * `vagrant up`
 
+#### Using the backup and restore scripts
+The scripts at the ~/backup_restore directory can be used to test backing up and restoring the fedora repository for consistency.
+
+The following command will cause 50 parallel processes to load data to the repository while creating snapshots of fcrepo home directory every 2 seconds.
+
+```
+cd ~/backup_restore/
+./hot_backup_runner.sh 50 /var/lib/tomcat7/fcrepo4-data 2
```

Comment:
When I run this command, after a few minutes I run out of disk space with:
`cp: cannot create directory ‘./backups/fcrepo_home.bkp.20151213181231’: No space left on device`
How much disk is needed to run this? Is there a way of specifying or limited the required disk space?